mods/ra/maps/allies-05a/allies05a.lua

```diff
@@ -131,19 +131,21 @@ ExtractUnits = function(extractionUnit, pos, after)
 end
 
 SendWaterExtraction = function()
-	local flare = Actor.Create("flare", true, { Owner = greece, Location = SpyEntryPath[2] + CVec.New(2, 0) })
+	---@type cpos
+	local path = SpyEntryPath[2]
+	local flare = Actor.Create("flare", true, { Owner = Greece, Location = path + CVec.New(2, 0) })
```

Comment:
Tbh not sure if we want to "fix" it like this, is there no way to make this work by default?